nibe/data/s1155_s1255.csv

```diff
@@ -1171,45 +1184,56 @@ id:6920	MODBUS_HOLDING_REGISTER	4074	1		4	0	20	0
 Blocking actions (ERS 4)	MODBUS_HOLDING_REGISTER	4101	1		4	0	2	2
-id:7048	MODBUS_HOLDING_REGISTER	3987	1		4	0	1	0
-id:7138	MODBUS_HOLDING_REGISTER	4009	1	%	4	0	100	70
-id:7139	MODBUS_HOLDING_REGISTER	4010	1		4	0	1	0
-id:7140	MODBUS_HOLDING_REGISTER	4011	1		4	0	1	0
-id:7141	MODBUS_HOLDING_REGISTER	4012	1	%	4	0	100	70
-id:7142	MODBUS_HOLDING_REGISTER	4013	1		4	0	1	0
-id:7143	MODBUS_HOLDING_REGISTER	4014	1		4	0	1	0
-id:7144	MODBUS_HOLDING_REGISTER	4015	1	%	4	0	100	70
-id:7145	MODBUS_HOLDING_REGISTER	4016	1		4	0	1	0
-id:7146	MODBUS_HOLDING_REGISTER	4017	1		4	0	1	0
-id:7147	MODBUS_HOLDING_REGISTER	4018	1	%	4	0	100	70
-id:7148	MODBUS_HOLDING_REGISTER	4019	1		4	0	1	0
-id:7149	MODBUS_HOLDING_REGISTER	4020	1		4	0	1	0
-id:7150	MODBUS_HOLDING_REGISTER	4021	1	%	4	0	100	70
-EB103/104-GP12	MODBUS_HOLDING_REGISTER	4022	1		4	0	1	0
-id:7176	MODBUS_HOLDING_REGISTER	4023	1		4	0	1	0
-id:8002	MODBUS_HOLDING_REGISTER	1996	1	%	4	-15	10	-3
+id:7048	MODBUS_HOLDING_REGISTER	5227	1		4	0	1	0
+id:7122	MODBUS_INPUT_REGISTER	2401	10	V	1	0	0	0
+id:7123	MODBUS_INPUT_REGISTER	2402	10	V	1	0	0	0
+id:7124	MODBUS_INPUT_REGISTER	2403	10	V	1	0	0	0
+id:7125	MODBUS_INPUT_REGISTER	2404	10	V	1	0	0	0
+id:7126	MODBUS_INPUT_REGISTER	2405	10	V	1	0	0	0
+id:7127	MODBUS_INPUT_REGISTER	2406	10	V	1	0	0	0
+id:7128	MODBUS_INPUT_REGISTER	2407	10	V	1	0	0	0
+id:7130	MODBUS_HOLDING_REGISTER	5433	1		4	0	1	0
+id:7133	MODBUS_HOLDING_REGISTER	5436	1		4	0	1	0
+id:7138	MODBUS_HOLDING_REGISTER	5441	1	%	4	0	100	70
+id:7139	MODBUS_HOLDING_REGISTER	5442	1		4	0	1	0
+id:7140	MODBUS_HOLDING_REGISTER	5443	1		4	0	1	0
+id:7141	MODBUS_HOLDING_REGISTER	5444	1	%	4	0	100	70
+id:7142	MODBUS_HOLDING_REGISTER	5445	1		4	0	1	0
+id:7143	MODBUS_HOLDING_REGISTER	5446	1		4	0	1	0
+id:7144	MODBUS_HOLDING_REGISTER	5447	1	%	4	0	100	70
+id:7145	MODBUS_HOLDING_REGISTER	5448	1		4	0	1	0
+id:7146	MODBUS_HOLDING_REGISTER	5449	1		4	0	1	0
+id:7147	MODBUS_HOLDING_REGISTER	5450	1	%	4	0	100	70
+id:7148	MODBUS_HOLDING_REGISTER	5451	1		4	0	1	0
+id:7149	MODBUS_HOLDING_REGISTER	5452	1		4	0	1	0
+id:7150	MODBUS_HOLDING_REGISTER	5453	1	%	4	0	100	70
+EB103/104-GP12	MODBUS_HOLDING_REGISTER	5454	1		4	0	1	0
+id:7176	MODBUS_HOLDING_REGISTER	5455	1		4	0	1	0
+id:7943	MODBUS_INPUT_REGISTER	2399	10	%	5	0	0	0
+id:7944	MODBUS_INPUT_REGISTER	2400	10	%	5	0	0	0
+id:8002	MODBUS_HOLDING_REGISTER	1996	1	%	1	-15	10	-3
 id:8003	MODBUS_HOLDING_REGISTER	1997	1	%	4	1	254	65
-Minimum permitted speed (EB100 GP1)	MODBUS_HOLDING_REGISTER	3242	1	%	4	1	50	1
-id:8060	MODBUS_HOLDING_REGISTER	3259	1		1	0	1	0
-id:8061	MODBUS_HOLDING_REGISTER	3260	1		1	0	1	0
-id:8062	MODBUS_HOLDING_REGISTER	3261	1		1	0	1	0
-id:8063	MODBUS_HOLDING_REGISTER	3262	1		1	0	1	0
-id:8064	MODBUS_HOLDING_REGISTER	3263	1		1	0	1	0
-id:8065	MODBUS_HOLDING_REGISTER	3264	1		1	0	1	0
-id:8066	MODBUS_HOLDING_REGISTER	3265	1		1	0	1	0
-id:8067	MODBUS_HOLDING_REGISTER	3266	1		1	0	1	0
+Minimum permitted speed (EB100 GP1)	MODBUS_HOLDING_REGISTER	5351	1	%	4	1	50	1
```

Comment:
@elupus I do not understand why these register addresses differ so much. Maybe we need to split some models?

Reply:
No idea. But they are unnamed registers soni think we can ignore them for the most part. Dont think we need to split.